server/routerlicious/pnpm-lock.yaml

```diff
@@ -1531,6 +1531,9 @@ importers:
       querystring:
         specifier: ^0.2.0
         version: 0.2.0
+      serialize-error:
```

Comment:
Wondering if this is needed or got added by mistake? I dont see it being used anywhere in this PR

Reply:
Fixed it..thank you for catching that